### PR TITLE
Feature/launch spoke

### PR DESCRIPTION
**Pull request by gfodor**
This is the PR we should land on Spoke launch day. It:

- Updates callouts to slack to point to Discord
- Adds the Spoke link to the top of the Hubs homepage
- Adds the "Made in Spoke" link to the bottom right of the scene page

### CODE REVIEW

File: src/react-components/home-root.js [hunk 1]
```diff
@@ -184,6 +184,12 @@ class HomeRoot extends Component {
                   <a href="https://github.com/mozilla/hubs" rel="noreferrer noopener">
                     <FormattedMessage id="home.source_link" />
                   </a>
+                  <a href="/spoke" rel="noreferrer noopener">
+                    <FormattedMessage id="home.spoke_link" />
+                  </a>
+                  <a href="https://discord.gg/XzrGUY8" rel="noreferrer noopener">
+                    <FormattedMessage id="home.community_link" />
+                  </a>
```
Comment (brianpeiris) [inline src/react-components/home-root.js]: This link is covered by this seemingly unused `ident` div:

image

File: src/react-components/home-root.js [hunk 2]
```diff
@@ -184,6 +184,12 @@ class HomeRoot extends Component {
                   <a href="https://github.com/mozilla/hubs" rel="noreferrer noopener">
                     <FormattedMessage id="home.source_link" />
                   </a>
+                  <a href="/spoke" rel="noreferrer noopener">
+                    <FormattedMessage id="home.spoke_link" />
+                  </a>
+                  <a href="https://discord.gg/XzrGUY8" rel="noreferrer noopener">
```
Comment (brianpeiris) [inline src/react-components/home-root.js]: We use this discord.gg link in a ton of places now. Might be nice to make a https://hubs.mozilla.com/community redirect in reticulum at somepoint

File: src/scene-entry-manager.js [hunk 3]
```diff
@@ -82,9 +83,16 @@ export default class SceneEntryManager {
     cursor.enable();
     cursor.setCursorVisibility(true);
 
-    this.hubChannel.sendEntryEvent().then(() => {
-      this.store.update({ activity: { lastEnteredAt: new Date().toISOString() } });
-    });
+    // Delay sending entry event telemetry until VR display is presenting.
```
Comment (brianpeiris) [inline src/scene-entry-manager.js]: Why are we doing this?

**Comment by brianpeiris** [comment]
Once our releases are not marked "pre-release", this will point to the latest release, maybe we should use it instead of pointing to all releases: https://github.com/MozillaReality/Spoke/releases/latest

File: src/scene-entry-manager.js (hunk 3 above)
Comment (gfodor) [reply src/scene-entry-manager.js]: see: https://github.com/mozilla/hubs/pull/618

**Pull request merged**